steel/src/lib.rs

```diff
@@ -37,10 +40,14 @@ mod state;
 pub use beacon::BeaconInput;
 pub use block::BlockInput;
 pub use contract::{CallBuilder, Contract};
-pub use history::HistoryInput;
 pub use mpt::MerkleTrie;
 pub use state::{StateAccount, StateDb};
 
+#[cfg(feature = "unstable-history")]
+pub use history::HistoryInput;
+#[cfg(not(feature = "unstable-history"))]
+pub(crate) use history::HistoryInput;
```

Comment:
I don't think there is an easy way to make EvmInput::History private, but since no such input can be created by the public API, and since it contains only private fields, this should be fine.

Reply:
Agreed